Cannot npm install after freshly cloning

**I'm submitting a ...**
```
[x] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
I hadn't removed node_modules and re-"npm install"d in a while (I've updated node since initially npm installing on this machine).  Attempted to do so and today and hit a compilation error during the post-install steps.  At first, I thought it might have been something I changed, so I cloned the seed fresh and received the same error (below).
```
$ npm install
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated express@2.5.11: express 2.x series is deprecated
npm WARN deprecated connect@1.9.2: connect 1.x series is deprecated
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'np
m ls graceful-fs' to find it in the tree.
npm WARN prefer global marked@0.3.6 should be installed with -g
npm WARN prefer global live-server@1.1.0 should be installed with -g
npm WARN prefer global node-gyp@3.6.2 should be installed with -g

> node-sass@4.5.3 install C:\Users\Isaac\Documents\angular-seed\node_modules\node-sass
> node scripts/install.js

Cached binary found at C:\Users\Isaac\AppData\Roaming\npm-cache\node-sass\4.5.3\win32-x64-57_binding.node

> node-sass@4.5.3 postinstall C:\Users\Isaac\Documents\angular-seed\node_modules\node-sass
> node scripts/build.js

Binary found at C:\Users\Isaac\Documents\angular-seed\node_modules\node-sass\vendor\win32-x64-57\binding.node
Testing binary
Binary is fine

> angular-seed@1.0.0 postinstall C:\Users\Isaac\Documents\angular-seed
> gulp check.versions && gulp build.bundle.rxjs && gulp webdriver && gulp print.banner

[21:34:45] Requiring external module ts-node/register

C:\Users\Isaac\Documents\angular-seed\node_modules\ts-node\src\index.ts:307
        throw new TSError(formatDiagnostics(diagnosticList, cwd, ts, lineOffset))
              ^
TSError: ⨯ Unable to compile TypeScript
tools\utils\seed\clean.ts (24,32): Property 'bgRed' does not exist on type 'typeof "C:/Users/Isaac/Documents/angular-seed/node_modules/@types/gulp-util/node_modules/chalk/ty...'. (2339)
tools\utils\seed\clean.ts (31,47): Property 'yellow' does not exist on type 'typeof "C:/Users/Isaac/Documents/angular-seed/node_modules/@types/gulp-util/node_modules/chalk/ty...'. (2339)
tools\utils\seed\clean.ts (39,40): Property 'red' does not exist on type 'typeof "C:/Users/Isaac/Documents/angular-seed/node_modules/@types/gulp-util/node_modules/chalk/ty...'. (2339)
    at getOutput (C:\Users\Isaac\Documents\angular-seed\node_modules\ts-node\src\index.ts:307:15)
    at C:\Users\Isaac\Documents\angular-seed\node_modules\ts-node\src\index.ts:336:16
    at Object.compile (C:\Users\Isaac\Documents\angular-seed\node_modules\ts-node\src\index.ts:498:11)
    at Module.m._compile (C:\Users\Isaac\Documents\angular-seed\node_modules\ts-node\src\index.ts:392:43)
    at Module._extensions..js (module.js:635:10)
    at Object.require.extensions.(anonymous function) [as .ts] (C:\Users\Isaac\Documents\angular-seed\node_modules\ts-node\src\index.ts:395:12)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Module.require (module.js:568:17)

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Isaac\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v8.5.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angular-seed@1.0.0 postinstall: `gulp check.versions && gulp build.bundle.rxjs && gulp webdriver && gulp print.banner`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular-seed@1.0.0 postinstall script 'gulp check.versions && gulp build.bundle.rxjs && gulp webdriver && gulp print.banner'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angular-seed package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp check.versions && gulp build.bundle.rxjs && gulp webdriver && gulp print.banner
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angular-seed
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angular-seed
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Isaac\AppData\Roaming\npm-cache\_logs\2017-10-25T01_34_46_434Z-debug.log
```
**Expected behavior**
dependencies installed successfully

**Minimal reproduction of the problem with instructions**
git clone https://github.com/mgechev/angular-seed.git
npm install

**What is the motivation / use case for changing the behavior?**
Using angular-seed with updated version of node?

**Please tell us about your environment:**
Windows 10

VS Code version 1.17.2

$ node -v
v8.5.0

$ npm -v
4.2.0

* **Angular Seed Version:** `d6abbf9`


* **Node:** `v8.5.0`

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "@types/gulp-load-plugins": "^0.0.30",
     "@types/gulp-protractor": "^1.0.30",
     "@types/gulp-sass": "^0.0.30",
-    "@types/gulp-util": "^3.0.29",
+    "@types/gulp-util": "3.0.31",
     "@types/jasmine": "^2.6.0",
     "@types/node": "^8.0.34",
     "@types/rimraf": "2.0.2",

